schema/ridl/parser_node.go

```diff
@@ -0,0 +1,348 @@
+package ridl
```

Comment:
This is an improved parser tree, it makes it easier to represent RIDL in terms of tokens (which will give us improved error messages) and to translate it to other formats, such as the WebRPC AST.